apps/web/src/pages/templates/components/ListProviders.tsx

```diff
@@ -17,6 +21,9 @@ export const ListProviders = ({
   setProvider: (provider: IIntegratedProvider) => void;
 }) => {
   const { colorScheme } = useMantineColorScheme();
+  const isMultiProviderConfigurationEnabled = !useIsMultiProviderConfigurationEnabled();
+  const { environments } = useFetchEnvironments();
+  const { environment: currentEnvironment } = useEnvController();
```

Comment:
Can we use only the useEnvController now?

Reply:
No funny enough useEnvController only return current env